seylii/templates/liiweb/home.html

```diff
@@ -0,0 +1,138 @@
+{% extends "liiweb/home.html" %}
+{% load static i18n %}
+{% block title %}
+  {% trans 'Home' %}
+{% endblock %}
+{% block hero-headings %}
+  <h1 class="mb-4">{% trans 'Welcome to the Seychelles Legal Information Institute' %}</h1>
+{% endblock %}
+{% block hero-logos %}
+  <div class="d-flex justify-content-between">
+    <img height="26"
+         class="me-4"
+         src="{% static 'images/africanlii-logo.svg' %}"
+         alt="africanlii-logo"/>
+    <img height="26"
+         src="{% static 'images/laws-africa-logo.png' %}"
+         alt="laws-logo"/>
+    <img height="26"
+         class="me-4"
+         src="{% static 'images/indigo-logo.jpg' %}"
+         alt="indigo-logo"/>
+    <img height="26"
+         class="me-4"
+         src="{% static 'images/judiciary-of-the-seychelles-logo.jpg' %}"
+         alt="judiciary-of-the-seychelles-logo"/>
+  </div>
+{% endblock %}
+{% block first-column-content %}
+  <section class="pb-3 pt-3">
+    <h2 class="mb-4 heading-underlined">{% trans 'Recent Judgments' %}</h2>
+    {% trans 'View more judgments' as text %}
+    {% include 'peachjam/_recent_document_list.html' with documents=recent_judgments view_more_text=text view_more_url_name='judgment_list' %}
+  </section>
+  <section class="pb-3 pt-3">
+    <h2 class="mb-4 heading-underlined">{% trans 'Recent Legislation' %}</h2>
+    {% trans 'View more legislation' as text %}
+    {% include 'peachjam/_recent_document_list.html' with documents=recent_legislation view_more_text=text view_more_url_name='legislation_list' %}
+  </section>
+{% endblock %}
+{% block second-column-content %}
+  <div>
```

Comment:
@nickmwangemi The pull requests I've just merges pulls this section into the base template, you don't need to add the social media tabs now. There are some blocks that you can reuse. Rebase the branch and check them out.